client/browser/src/browser-extension/options-menu/OptionsPage.tsx

```diff
@@ -263,7 +263,7 @@ const RepoSyncErrorAlert: React.FunctionComponent<
 const SourcegraphCloudAlert: React.FunctionComponent<React.PropsWithChildren<unknown>> = () => (
     <section className={classNames('bg-2', styles.section)}>
         <h4>
-            <Icon className="mr-2" as={CheckCircleOutlineIcon} />
+            <Icon aria-hidden={true} role="img" className="mr-2" as={CheckCircleOutlineIcon} />
```

Comment:
```suggestion
            <Icon role="img" aria-hidden={true} className="mr-2" as={CheckCircleOutlineIcon} />
```

So we can easily find + replace the `role="img"` part when we set this as the default for `Icon`